boot/freeldr/freeldr/ntldr/wlregistry.c

```diff
@@ -141,7 +141,7 @@ WinLdrInitSystemHive(
     }
 
     /* Import what was loaded */
-    Success = RegImportBinaryHive(VaToPa(LoaderBlock->RegistryBase), LoaderBlock->RegistryLength);
+    Success = RegImportBinaryHive(LoaderBlock);
```

Comment:
This was originally done such that registry.c doesn't depend on the rest of definitions needed for the whole LOADER_PARAMETER_BLOCK structure. So please revert.
Also, what happens to the `RegistryLength` value now?